src/component/description/cases/observing/Observe.js

```diff
@@ -25,6 +26,14 @@ export class Observe extends Component {
     dispatchHeaderBackground({ newColor });
   }
 
+  handleLang = (lang) => {
+    const newLang = lang.value;
+    const {
+      dispatchDefaultLanguage,
```

Comment:
Again, `dispatchDefaultLanguage` is not the right name. It should be `dispatchChangeLanguage`.